components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md

```diff
@@ -94,39 +94,52 @@ This page explains migrating the existing A2HA data to the newly deployed Chef A
 
 1. Add the OpenSearch credentials to the applied config.
 
-    - If using Chef Managed Opensearch, then add the below config into `current_config.toml` (without any changes).  
+    - If using Chef Managed Opensearch, add the config below into `current_config.toml` (without any changes).  
 
         ```bash
         [global.v1.external.opensearch.auth.basic_auth]
             username = "admin"
             password = "admin"
         ```
 
+    - In case of ``` backup_config = "file_system" ``` had been provided in config.toml of Automate HA deployment, then need to patch below OpenSearch config from bastion before starting the restore.
```

Comment:
```suggestion
    - In case ` backup_config = "file_system"` had been provided in config.toml of Automate HA deployment, then please patch below OpenSearch config from bastion before starting the restore.
```